src/components/atoms/Counter.svelte

```diff
@@ -5,13 +5,13 @@
   // Import fonts, vars, etc.
   @import 'src/styles/index.scss';
 
-  h1 {
+  h2 {
     font-size: 2em;
     color: $ui-black;
   }
 
 </style>
 
-<h1>
+<h2>
```

Comment:
I would still make a p tag of this. Because we also already have a subtitle which is an h2. So no titles in this area 😄